src/runtime/components/NuxtTurnstile.vue

```diff
@@ -32,31 +32,38 @@ const config = useRuntimeConfig().public.turnstile
 const nuxtApp = useNuxtApp()
 
 const el = ref()
+const unmountStarted = ref(false)
 
 let interval: NodeJS.Timeout
 
-function reset() {
-  return nuxtApp.$turnstile.reset(el.value)
+const reset = () => nuxtApp.$turnstile.reset(el.value)
+const unmount = () => {
+  unmountStarted.value = true
+  clearInterval(interval)
+  nuxtApp.$turnstile.rm(el.value)
 }
 
-defineExpose({ reset })
-
 onMounted(async () => {
-  await nextTick()
```

Comment:
@danielroe any reason this `nextTick` is necessary? Everything seems to work without it as well.

Reply:
I think likely added because of https://github.com/nuxt/framework/issues/3587.

Reply:
so should we keep it in case a `NuxtTurnstile` get wrapped within a `transition` ? (i wasn't aware about this issue)

Reply:
I'll add it back in and add a comment that links to the issue so future visitors understand the need for this workaround right away :raised_hands: